README.md

```diff
@@ -1,15 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My-react-app
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- REACT.
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] [OPTIONAL] Please make sure to edit each section of the readme to better suit your project.